Formula/makensis.rb

```diff
@@ -3,6 +3,7 @@ class Makensis < Formula
   homepage "https://nsis.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/nsis/NSIS%203/3.04/nsis-3.04-src.tar.bz2"
   sha256 "609536046c50f35cfd909dd7df2ab38f2e835d0da3c1048aa0d48c59c5a4f4f5"
+  revision 1 
```

Comment:
`C:  6: 13: Trailing whitespace detected.`

Reply:
Sure enough. Thanks. Fixed in https://github.com/Homebrew/homebrew-core/pull/43558/commits/b758497666f6f97c73fd9d3a8e0db73707d7185c